src/ontology/cl-edit.owl

```diff
@@ -30074,6 +30077,30 @@ AnnotationAssertion(rdfs:label obo:CL_4030008 "pronephric podocyte")
 EquivalentClasses(obo:CL_4030008 ObjectIntersectionOf(obo:CL_0000653 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002120)))
 SubClassOf(obo:CL_4030008 obo:CL_0000653)
 
+# Class: obo:CL_4030009 (epithelial cell of proximal tubule segment 1)
```

Comment:
@bvarner-ebi 
I agree that it's hard to get these long labels 100% right. Personally I'd prefer 'epithelial cell of segment 1 of proximal tubule' for complete disambiguation, but it makes for a longer label. Another option is to leave 'epithelial cell of proximal tubule segment 1' but add an exact synonym "epithelial cell of segment 1 of proximal tubule". Same for the 2 sibling terms. Thanks.

Reply:
Thank you, @paolaroncaglia!

There seems to be inconsistency among preexisting labels... some are structured

'% cell of %', 

whilst others are of the format 

'% cell'

Examples:
'epithelial cell of proximal tubule',
'brush border cell of the proximal tubule',
'kidney proximal convoluted tubule epithelial cell', 
'kidney inner medulla collecting duct epithelial cell'

I'll add synonyms as you suggest in the latter half of your comment.

Reply:
@bvarner-ebi ok with me, thanks!